bearer/bearer.go

```diff
@@ -253,8 +253,8 @@ func (b Token) AssertUser(id user.ID) bool {
 // Note that any Token mutation is likely to break the signature, so it is
 // expected to be calculated as a final stage of Token formation.
 //
-// See also VerifySignature, Issuer.
-func (b *Token) Sign(signer neofscrypto.Signer) error {
+// See also VerifySignature, Issuer, [Token.SignedData].
+func (b *Token) Sign(signer user.Signer) error {
```

Comment:
dont do this just for test, make a workaround for the test if needed

Reply:
In this case we should duplicate `SignedComponent` for `neofscrypto.Signer` and for `user.Signer`

Reply:
But why? This is certainly not worth a change for test. And tests should be fine with `neofscrypto.Signer` since `user.Signer` includes it.